packages/accounts-base/accounts_client.js

```diff
@@ -1,51 +1,132 @@
-// @summary Constructor for AccountsClient instances. Available only on
-//          the client for now, though server code might also want to
-//          create a client connection to an accounts server.
-// @locus Client
-// @param options {Object} an object with fields:
-// - connection {Object} Optional DDP connection to reuse.
-// - ddpUrl {String} Optional URL for creating a new DDP connection.
-AccountsClient = function _AccountsClient(options) {
-  AccountsCommon.call(this, options);
-
-  this._loggingIn = false;
-  this._loggingInDeps = new Tracker.Dependency;
-
-  this._loginServicesHandle =
-    this.connection.subscribe("meteor.loginServiceConfiguration");
+/**
+ * @summary Constructor for the `Accounts` namespace on the client.
```

Comment:
Oh, nvm, saw the note: "Note that AccountsClient is currently available only on the client, due to its use of browser APIs such as window.localStorage. In principle, though, it might make sense to establish a client connection from one server to another remote accounts server. Please let us know if you find yourself needing this server-to-server functionality."

Great explanation.